Reference/caves.json

```diff
@@ -79,7 +79,7 @@
   {
     "type": "cave",
     "name": "Avos Animation Totem",
-    "specialInfo": "Boss Cave",
+    "specialInfo": "§eBoss Cave",
```

Comment:
Why do you introduce formatting here? I tried to check if it is ever anything else than `§e` but I could not see that. If that is indeed correct, I don't think we should have formatting in the json data, but add it in the mod if needed.

Reply:
Basically the same answer as above :). I only care about this file for the updated coordinates, hopefully it'll be deprecated soon, and never seen again :)

Reply:
I see, I had forgotten about the specifics here. Thanks!